boxoffice/mailclient.py

```diff
@@ -55,15 +55,19 @@ def send_line_item_cancellation_mail(line_item_id, subject="Ticket Cancellation"
 
 
 @job('boxoffice')
-def send_ticket_assignment_mail(line_item_id):
+def send_ticket_assignment_mail(line_item_id, past_assignee):
     """
     Sends a confirmation email once details are filled and ticket has been assigned.
     """
     with app.test_request_context():
         line_item = LineItem.query.get(line_item_id)
         order = line_item.order
         subject = order.item_collection.title + ": Here's your ticket"
-        msg = Message(subject=subject, recipients=[line_item.current_assignee.email], bcc=[order.buyer_email])
+        if past_assignee:
+          recipient_list = [order.buyer_email, past_assignee]
```

Comment:
This should be `cc_list` since recipient would imply the intended recipient of the email.